applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -611,6 +670,13 @@ The application supports the following configurations:
      - Release version of the application.
 
        Enables only the necessary application functionalities to optimize its performance.
+   * - Matter bridge over Wi-Fi
```

Comment:
```suggestion
   * - Matter bridge over Wi-Fi with nRF5340 DK and nRF7002 EK
```

Reply:
I don't think the devices need to be in the configuration name, in case we add more supported devices later.

Reply:
But the thing is that there are already more devices that support Matter bridge over Wi-Fi and this particular configuration is related only to nrf5340dk+nrf70ek. Other configurations like nrf54h20+nrf70ek or nrf7002dk are built using other way.

Reply:
Ah, I missed that part. In that case, it makes since to keep the device mentions in both places. 

```suggestion
   * - Matter bridge over Wi-Fi with the nRF5340 DK and nRF7002 EK
```
Note the definitive article.